napari/layers/image/image.py

```diff
@@ -373,6 +374,7 @@ def __init__(
         self.depiction = depiction
         if plane is not None:
             self.plane = plane
+        self.plane.events.connect(self._plane_event)
```

Comment:
Would 
```suggestion
        self.plane.events.connect(self.events.plane)
```

Not work here? Just trying to avoid the dummy method if possible

Reply:
Mhm, that's strange, that was literally the first thing I tried and I got a whole bunch of errors which seemed genuine... Now if I do that it just works. @Czaki any clue if this has some issues?

Reply:
There may be an issue connected with passing the original event to new event emitter, but it could be solved by using `connect_no_arg`: 
https://github.com/napari/napari/blob/8bb7e78f8fa9ec05fa82302c9e27254a85c1cc11/napari/utils/events/event_utils.py#L45-L51

Reply:
Sorry, I'm not following. Can you give me an example of what you mean?

Reply:
```python
connect_no_arg(self.plane.events, self.events, "plane")
```
with proper import